src/main/java/io/cryostat/rules/PeriodicArchiver.java

```diff
@@ -150,7 +150,9 @@ private void performArchival() throws InterruptedException, ExecutionException,
     }
 
     private void pruneArchive(String recordingName) throws Exception {
-        recordingArchiveHelper.deleteRecording(recordingName).get();
+        recordingArchiveHelper
+                .deleteRecording(serviceRef.getServiceUri().toString(), recordingName)
```

Comment:
For a potential future refactor, should this first `String targetId/connectUrl` parameter be replaced with the whole `ServiceRef` to signify the sourceTarget? Only the connectUrl or jvmId is probably actually relevant, but it gives the option to use either of those as well as provides some compile-time type safety to guard against accidents like this one.